compiler/src/dmd/dimport.d

```diff
@@ -232,7 +232,21 @@ extern (C++) final class Import : Dsymbol
          * most likely because of parsing errors.
          * Therefore we cannot trust the resulting AST.
          */
-        if (load(sc)) return;
+        import core.stdc.stdio;
```

Comment:
Put this at the top of the module for future debug printfs

Reply:
sorry, I thought I deleted it.

Reply:
better still don't include it at all

Reply:
Walter explicitly asked in an email to keep `import core.stdc.stdio;` imports around for debug `printf`s.